app/helpers/date_helper.rb

```diff
@@ -102,7 +102,7 @@ def join_time_select_values(values)
   private
 
   def minute_options(step = nil)
-    step ||= 5
+    step ||= 1
```

Comment:
Should we make the default depend on the instrument pricing mode? I think for the non-Daily booking instruments it should be 5

Reply:
We could add it elsewhere so this helper does not depend on instruments. I took that in mind but since non daily booking instruments have the field reserve_interval as required then this value should be set for those

Reply:
This is only used in the view, right? In that case, I think it's better the solution you had before, setting it in the model callback. Sorry I didn't understand it properly the first time